src/Common/IOperationExtensions.cs

```diff
@@ -18,4 +18,43 @@ public static IOperation WalkDownConversion(this IOperation operation)
 
         return operation;
     }
+
```

Comment:
nit: For the stuff that's marked public, and hence intended to be reusable across the project, can we do xmldocs to explain what they do, examples, whatever?

Reply:
I'll follow up with a different PR. Most things should be internal (`internal` is my visibility default). There's only a few dozen things that actually need to be public (e.g., the analyzers and code fixers and some common stuff shared); everything else should be internal at best